lib/Sema/MiscDiagnostics.cpp

```diff
@@ -1328,7 +1328,26 @@ void swift::fixItAvailableAttrRename(TypeChecker &TC,
     return labelStr.empty() ? Identifier() : TC.Context.getIdentifier(labelStr);
   });
 
-  if (auto args = dyn_cast<TupleExpr>(argExpr)) {
+  if (auto args = dyn_cast<TupleShuffleExpr>(argExpr)) {
+      // FIXME: Can we really do this?
+      argExpr = args->getSubExpr();
+  }
+
+  if (auto args = dyn_cast<ParenExpr>(argExpr)) {
```

Comment:
In particular, the else clause is already supposed to cover all the ParenExpr cases. I suppose it should assert that, since clearly I forgot about TupleShuffleExpr.